integreat_cms/cms/templates/hix_widget.html

```diff
@@ -11,7 +11,7 @@
 {% block collapsible_box_content %}
     <div class="relative">
         <div id="hix-loading"
-             class="hidden absolute w-full h-full bg-white bg-opacity-75">
```

Comment:
I guess this means that the following line can be deleted, right?
https://github.com/digitalfabrik/integreat-cms/blob/9b4fff616594ca995a9d24b79d74a3c0caf6ae3d/integreat_cms/static/src/js/analytics/hix-widget.ts#L98